src/SimpleThings/EntityAudit/EventListener/LogRevisionsListener.php

```diff
@@ -326,7 +326,8 @@ private function getInsertRevisionSQL($class)
             }
 
             foreach ($class->fieldNames as $field) {
-                if (array_key_exists($field, $fields)) {
+                $columnName = $class->fieldMappings[$field]['columnName'];
```

Comment:
That's what we understood, we did not try to argue that it was more or less "ready" :trollface: